python/cuml/metrics/base.py

```diff
@@ -32,8 +32,8 @@ def score(self, X, y, **kwargs):
         ----------
         X : [cudf.DataFrame]
             Test samples on which we predict
-        y : [cudf.Series]
-            True values for predict(X)
+        y : [cudf.Series, device array, or numpy array]
+            Ground truth values for predict(X)
```

Comment:
Maybe we could change this docstring to be the normal format, or maybe we should open an issue to go around our docstrings to check uniformity? What do you think?

Reply:
I'll open an issue. Here, there were just too many docstring changes to roll into this change, so I kept it locally consistent.